tests/chainer_tests/test_variable.py

```diff
@@ -371,28 +371,37 @@ def test_label_gpu(self):
         self.check_label(self.label, cuda.to_gpu(self.c))
 
     def check_backward(self, inputs, intermediates, outputs, retain_grad):
-        for o in outputs:
-            o.backward(retain_grad)
+        # This test assumes that `outputs` do not depend each other
```

Comment:
How about writing a brief summary of this test?
(I think it's not easily guessed.)